Makefile

```diff
@@ -2,7 +2,7 @@ CWD = $(shell pwd)
 SRC_DIR = ${CWD}/pootle
 DOCS_DIR = ${CWD}/docs
 STATIC_DIR = ${SRC_DIR}/static
-ASSETS_DIR = $(shell python -c "from pootle.settings import *; print(STATIC_ROOT)")
+ASSETS_DIR = ${SRC_DIR}/assets
```

Comment:
I added the comment to the Makefile as well (though it looks lonely and won't likely serve any purpose). The reason i believe the comment in the configuration file makes more sense (and should be kept) is that though we expose the STATIC_ROOT in the config file and technically allow to overwrite it, end users shouldn't be doing this arbitrarily, and the comment gives guidance on when and how the changes make sense, and what implications the changes would have. Since `make assets` is shipped with Zing and is used for deployment, it is not that 'optional', and deserves being mentioned in the comment. If at some point we render `make assets` unnecessary, or remove the STATIC_ROOT from settings altogether, this will render my comment irrelevant. Before that, I'd prefer to keep it there.